ed_sensor_integration/src/laser/laser_plugin.h

```diff
@@ -33,8 +41,10 @@ class LaserPlugin : public ed::Plugin
 
     virtual ~LaserPlugin();
 
+    // initialise plugin
```

Comment:
If I'm being picky: these two functions are public, hence the API can/should be used by other parties. Hence, these are more important to describe properly than the private ones.

Reply:
I think in this case a docstring would not provide more information than the name of the function itself

Reply:
Question: why are these definitions here anyway? Shouldn't they be defined in the Plugin superclass since this is a template?

Reply:
Yes, these should be documented in the `ED::Plugin` class.